etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/PartnerMetadata.java

```diff
@@ -25,6 +25,15 @@ public PartnerMetadata(
         this(receivedSubmissionId, null, sender, null, timeReceived, hash);
     }
 
+    public PartnerMetadata(
```

Comment:
Will this be removed?

Reply:
Yes in the PR to add Submission received id

Reply:
So it will be added in this PR, and then removed in the other one? Is that necessary?